Synapse3.SynapseModule/Player/SynapsePlayerCommonProperties.cs

```diff
@@ -247,6 +247,49 @@ public SynapsePlayer CurrentlySpectating
         }
     }
 
+    public float SneakSpeed
+    {
+        get => FirstPersonMovement?.SneakSpeed ?? 0;
+        set
+        {
+            var firstperosn = FirstPersonMovement;
+            if (firstperosn != null)
+                firstperosn.SneakSpeed = value;
+        }
+    }
+
+    public float WalkSpeed
+    {
+        get => FirstPersonMovement?.WalkSpeed ?? 0;
+        set
+        {
+            var firstperosn = FirstPersonMovement;
+            if (firstperosn != null)
+                firstperosn.WalkSpeed = value;
+        }
+    }
+
+    public float RunSpeed
+    {
+        get => FirstPersonMovement?.SprintSpeed ?? 0;
+        set
+        {
+            var firstperosn = FirstPersonMovement;
+            if (firstperosn != null)
+                firstperosn.SprintSpeed = value;
+        }
+    }
+    public float CrouchingSpeed
+    {
+        get => FirstPersonMovement?.CrouchSpeed ?? 0;
+        set
+        {
+            var firstperosn = FirstPersonMovement;
+            if (firstperosn != null)
+                firstperosn.CrouchSpeed = value;
+        }
+    }
+
```

Comment:
It only sets the speed server side and affects the anti cheat but doesn't sync with the client

Reply:
yes but we can use it with a fake sync var, for the moment I don't know if it was already done so I didn't add it

Reply:
ho the prop MovementMultiplier existe ok